subprojects/platform-native/src/testFixtures/groovy/org/gradle/nativeplatform/fixtures/app/IncrementalSwiftXCTestRemoveDiscoveryBundle.groovy

```diff
@@ -19,44 +19,25 @@ package org.gradle.nativeplatform.fixtures.app
 import org.gradle.integtests.fixtures.SourceFile
 
 class IncrementalSwiftXCTestRemoveDiscoveryBundle extends IncrementalSwiftXCTestElement  {
-    private static final String PASSING_ASSERTION = "XCTAssert(true)"
     String moduleName = "RemoveDiscovery"
 
-    final XCTestSourceFileElement fooTestSuite = new XCTestSourceFileElement() {
-        String testSuiteName = "FooTestSuite"
+    final XCTestSourceFileElement fooTestSuite = new XCTestSourceFileElement("FooTestSuite") {
         List<XCTestCaseElement> testCases = [
-            testCase("testA", PASSING_ASSERTION),
-            testCase("testB", PASSING_ASSERTION)
+            passingTestCase("testA"),
```

Comment:
These new assertions are wrong. They should be reversed with the alternate as the fixture is for removing a test case.

Reply:
Good catch!